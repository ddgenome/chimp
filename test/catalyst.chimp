## catalytic mechanism, but don't use the kmc lattice
mechanism "catalyst.mech"
## parameter input
parameter "catalyst.par"
## catalyst task
task "catalyst.task"
