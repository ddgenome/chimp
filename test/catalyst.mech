# catalyst without lattice
# adsorption / desorption
A + 2 @ -> k(A_Aads) <- k_arrhenius(A_Ades, E_Ades) @@A;
# surface reaction
@@A -> k_lfer(A_srf, E0_sr, a_srf, DH_srf)
  <- k_lfer(A_srr, E0_sr, 1.0e0 - a_srf, -DH_srf) 2 @B;
@B -> k_arrhenius(A_Bdes, E_Bdes) <- k(A_Bads) B + @;
