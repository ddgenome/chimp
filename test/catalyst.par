# catalyst parameter input file
A_Aads	7.0e2
A_Ades	1.0e11
E_Ades	7.0e4
A_srf	1.0e12
E0_sr	6.1e4
a_srf	5.0e-1
DH_srf	-1.5e4
A_srr	1.0e12
A_Bdes	1.0e11
E_Bdes	6.5e4
A_Bads	6.0e-1
