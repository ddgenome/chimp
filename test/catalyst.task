# -*- text -*-
# test task input
begin model catalyst
  output "catalyst.out"
  begin integrator kmc
    scale 1.0e15
    begin state
      begin quantity
	p[A] = 1.0e5		# Pa
	@[@@A] = 1.0e-2
	@[@B] = 9.7e-1
      end quantity
      begin output
	(1.0e-2 1.0e-1 1.0e-2)
      end output
      begin reactor batch
	temperature 4.0e2	# K
	pressure 1.0e5		# Pa
	volume 1.0e-5		# m^3
	sites 1.0e19
	rate_numerator molecules
	rate_denominator sites
	fluid_quantity pressure
      end reactor
    end state
  end integrator
end model
