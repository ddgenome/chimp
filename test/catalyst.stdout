chimp 0.1
Copyright (C) 2004 David Dooling <banjo@users.sourceforge.net>

There is ABSOLUTELY NO WARRANTY; see file COPYING for details.
This is free software, and you are welcome to redistribute it
under certain conditions; see file COPYING for details.

Report bugs to http://sourceforge.net/projects/chimp/.
chimp: performing task catalyst.par... completed; 0u 0s 0w
chimp: performing task catalyst... completed; 4.54u 0s 4.54w
